src/components/link/Link.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './Link.scss';
+
+export default function Link({
+  href,
+  text,
+  external,
+}: {
+  href: string;
+  text: string;
+  external: boolean;
+}): JSX.Element {
+  return (
+    <a
+      href={href}
+      className="custom-link"
+      {...(external ? { rel: 'noopener noreferrer', target: '_blank' } : {})}>
```

Comment:
Semisti kryptinen tämä rivi :D. Mietin saisko tätä yksinkertaistettua jotenkin? Ilmeisesti tässä halutaan erotella se, että avautuuko linkin sivu kokonaan uuteen tabiin, vai pysytäänkö sovelluksen sisällä? Vois olla luettavampi siten, että kumpikin attribuutti (rel ja target) olis omilla riveillään, vaikkakin siitä tulee hieman toistoa

Reply:
I can try to make it a bit more readable 👍